<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="routing_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>601</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Destination Prefix</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Exit Interface</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next Hop</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>1061</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="arp_table_view">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>331</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MAC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TTL</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ARP for:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="arp_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="arp_name_2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="arp_table_view_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>331</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MAC</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TTL</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ARP for:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RIP next peridoic update in (s):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rip_timer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="rip_table">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>631</width>
        <height>201</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>prefix</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>source</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>holddown</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>invalid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>flush</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>state</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
